### PR TITLE
[skin] show playspeed

### PR DESCRIPTION
**Pull request by Rechi**
This shows the playspeed in the skins from #10216

### CODE REVIEW

File: addons/skin.estouchy/xml/Includes.xml [hunk 1]
```diff
@@ -75,6 +75,7 @@
 		<value condition="Player.Forwarding">$LOCALIZE[31044] $VAR[Speed]</value>
 		<value condition="Player.Rewinding">$LOCALIZE[31045] $VAR[Speed]</value>
 		<value condition="Player.Paused">[UPPERCASE]$LOCALIZE[31043][/UPPERCASE]</value>
+		<value condition="Player.IsTempo">$LOCALIZE[31046]: $INFO[Player.PlaySpeed]</value>
```
Comment (ronie) [inline addons/skin.estouchy/xml/Includes.xml]: please use [UPPERCASE] here

**Comment by Rechi** [comment]
@ronie updated

**Comment by FernetMenta** [comment]
great!

**Comment by Rechi** [comment]
I thought it might be useful to replace the condition `Player.IsTempo` with `Player.TempoEnabled + Player.Playing`.
Then it will also display `Play speed: 1.00`, but only if the player can change speed. This would show the ability of changing speed for the actual stream.

**Comment by FernetMenta** [comment]
Player.Playing is redundant in this context. TempoEnabled returns false if player is not active.

**Comment by Rechi** [comment]
TempoEnabled also returns true if player is fastforwarding or rewinding, but I think `Play speed: *` only should be displayed if playing.
My question was if it also makes sense to display `Play speed: 1.00` or should it display nothing in this case.

**Comment by FernetMenta** [comment]
right, this differs from the internal implementation where IsPlaying is also true for ff/rw.

> My question was if it also makes sense to display Play speed: 1.00 or should it display nothing in this case.

Good question. Now that we have a label for speed, does it still make sense to display "forwarding" or "rewinding"? Why not just show the speed label for all cases?

**Comment by MartijnKaijser** [comment]
@ronie @BigNoid @phil65 ?

**Comment by ronie** [comment]
tested and works ok at my end.

thx!

**Pull request merged**